nano/node/vote_processor.cpp

```diff
@@ -18,7 +18,7 @@ thread ([this]() {
 
 void nano::vote_processor::process_loop ()
 {
-	nano::timer<std::chrono::milliseconds> elapsed;
+	nano::timer<std::chrono::milliseconds> elapsed (nano::timer_state::stopped);
```

Comment:
A nano timer is initially stopped so passing the stopped state shouldn't be necessary. Otherwise LGTM.